.github/workflows/updates-e2e.yml

```diff
@@ -0,0 +1,82 @@
+name: Updates e2e
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/**
```

Comment:
The expo package is getting updated a lot (especially because of the CLI). We can exclude paths with `!packages/expo/cli/**` but what specifically are you looking to test from the expo package?

Reply:
My thinking was that since the expo package is a dependency of the build, changes to it could cause this job to break and it would be best to catch that at the source. But I'm fine with excluding the cli folder, or look specifically for changes in the ios/ and android/ directories